labs/firepower-restapi-111/2.md

```diff
@@ -3,44 +3,42 @@ Using Postman
 
 1.  Open your POSTMAN REST API Client.
 
-2.  Set the client to make a POST command to the following URL: https://cisco-tid.cisco.com/api/fmc_platform/v1/auth/generatetoken
+2.  Set the client to make a POST command to the this [URL](https://cisco-tid.cisco.com/api/fmc_platform/v1/auth/generatetoken).
 
-3.  Click on Authorization, select Basic Auth and type in your username and password.
+3.  Click on Authorization, select Basic Auth, and type in your username and password.
 
-4.  Click Send
+4.  Click Send.
 
  ![Figure: FMC REST API example setup](/posts/files/firepower-restapi-111/assets/images/Picture4.png)
 
+
 1.  From the response copy the following for future API calls:
-    >X-auth-access-token: You need to add this token header to every request sent
-    using FMC REST API
+  - X-auth-access-token: you need to add this token header to every request sent using FMC REST API.
 
-    > X-auth-refresh-token: is needed to refresh the token.
+  - X-auth-refresh-token: is needed to refresh the token.
 
-    > DOMAIN\_UUID: This needs to be added to every Rest API route.
+  - DOMAIN\_UUID: this needs to be added to every Rest API route.
 
 ![Figure: FMC REST API example setup](/posts/files/firepower-restapi-111/assets/images/Picture5.png)
 
-## Creating a TID Source:
 
-### To Create a TID Source Using URL Download:
+## Creating a TID source:
```

Comment:
We're trying to get rid of the gerunds and replace them with imperatives. In this case, it should be "Create a TID source". Also, there should be no colon in a header.

Reply:
Should the page title be Use Postman instead of Using Postman? I've changed it, since it seems like that's what you want everywhere, but let me know if the tile is okay with gerunds.